server/routerlicious/packages/services/src/throttler.ts

```diff
@@ -98,10 +107,12 @@ export class Throttler implements IThrottler {
             await this.throttlerHelper.updateCount(id, countDelta)
                 .then((throttlerResponse) => {
                     this.logger?.info(`Incremented throttle count for ${id} by ${countDelta}`, { messageMetaData });
+                    Lumberjack.info(`Incremented throttle count for ${id} by ${countDelta}`, { messageMetaData });
                     this.throttlerResponseCache.set(id, throttlerResponse);
                 })
                 .catch((err) => {
                     this.logger?.error(`Failed to update throttling count for ${id}: ${err}`, { messageMetaData });
+                    Lumberjack.error(`Failed to update throttling count for ${id}: ${err}`, { messageMetaData });
```

Comment:
```
Lumberjack.error(`Failed to update throttling count for ${id}`, lumberjackProperties, err);
```

You can create a common `lumberjackProperties` object in this function/file and use it for all Lumberjack calls instead of messageMetadata. Also, let's remove messageMetadata on line 102.